apps/portals/src/configurations/genie/synapseConfigs/dataReleases.ts

```diff
@@ -2,6 +2,21 @@ import { SynapseConfig } from 'types/portal-config'
 import { currentDataReleasesSql, dataReleasesSql } from '../resources'
 import { SynapseConstants } from 'synapse-react-client'
 
+// TODO: confirm names after columns are added to source table
+const sourcePathColumnName = 'releaseExplorePath'
+const sourceExploreDataSqlColumnName = 'exploreDataSql'
```

Comment:
Current GENIE release cards: 

<img width="1317" alt="PORTALS-3014_geniePortal" src="https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/26949006/0ee9fef2-2263-42f0-bfd5-b86d8f9ddd55">

The primary and secondary buttons will be shown and link to filtered explore pages after the [current release table](https://www.synapse.org/#!Synapse:syn53978783/tables/) is updated to: 
- Add columns: 
  -`releaseExplorePath`: "/Explore/GENIE" or "/Explore/GENIE-BPC"
  - `exploreDataSql`: "SELECT * FROM syn54100284" or "SELECT * FROM syn54100290" 
  - `version`: values matching those in tables driving explore pages ([GENIE](https://www.synapse.org/#!Synapse:syn54100284/tables/), [GENIE-BPC](https://www.synapse.org/#!Synapse:syn54100290/tables/))
- Ensure that values in "Cohort" column match values in tables driving explore pages ([GENIE](https://www.synapse.org/#!Synapse:syn54100284/tables/), [GENIE-BPC](https://www.synapse.org/#!Synapse:syn54100290/tables/)), so that facet selection works as expected

I've discussed the list of updates with the GENIE portal stakeholders and sent over an example table.